dao/endpoint_dao.go

```diff
@@ -14,11 +14,11 @@ type EndpointDaoImpl struct {
 
 func (a *EndpointDaoImpl) SubCollectionList(primaryCollection interface{}, limit int, offset int, filters []util.Filter) ([]m.Endpoint, int64, error) {
 	endpoints := make([]m.Endpoint, 0, limit)
-
 	sourceType, err := m.NewRelationObject(primaryCollection, *a.TenantID, DB.Debug())
 	if err != nil {
-		return nil, 0, err
+		return nil, 0, util.NewErrNotFound("source type")
```

Comment:
I think that this one is a "source" instead of a "source type".  The reason is that this comes from the "sources/:source_id/endpoints" route, which would list all the endpoints from a given source.

Reply:
yes i agree .. brilliant eye ... i checked it before and i thought i changed it but i missed it ... thanks